### ISSUE TITLE
chrome webstore manager has a vulnerability

### ISSUE THREAD

**Issue by aha-oretama** [pastak/chrome-webstore-manager#8]
I run `npm audit`. Then I found a vulnerability in this library.

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ open                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ chrome-webstore-manager [dev]                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ chrome-webstore-manager > open                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/663                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by pastak** [comment]
Oh, thanks for notice. I'll update

**Comment by pastak** [comment]
I have updated and released as `v0.4.2`
